scipy/interpolate/setup.py

```diff
@@ -41,9 +41,9 @@ def configuration(parent_package='',top_path=None):
     config.add_extension('_fitpack',
                          sources=['src/_fitpackmodule.c'],
                          libraries=['fitpack'],
-                         define_macros=define_macros,
```

Comment:
Ah, this is the problem. This `=define_macros` is not empty only in case of the ILP build, higher up it has:
```
define_macros = [("HAVE_ILP64", None)]
```
So you want to merge that with `numpy_nodepr_api` so both macros get defined. Something like:
```python
_fitpack_macros = define_macros + numpy_nodepr_api['define_macros']

config.add_extension('_fitpack',
    ....
   define_macros=_fitpack_macros
```

Reply:
Yeah, I will change that. Thanks!!